### PR TITLE
[PAY-724] Special Error Treatment and libs/sdk bump to 1.0.23

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

❗ ⚠️  depends on https://github.com/AudiusProject/audius-protocol/pull/4375

[PAY-724](https://linear.app/audius/issue/PAY-724/account-for-relay-403s-on-signup-analytics) for details

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested locally manually linking libs web chrome against stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-special-error

**Review by sddioulde — APPROVED** [review]
nice

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-special-error

File: packages/web/src/pages/something-wrong/SomethingWrong.tsx [hunk 1]
```diff
@@ -21,14 +23,22 @@ const messages = {
...
+  [UiErrorCode.UNKNOWN]: <i className='emoji xl heavy-black-heart' />,
+  [UiErrorCode.RELAY_BLOCKED]: <i className='emoji xl confused-face' />
+}
+
 export const SomethingWrong = () => {
   const lastLocation = useLastLocation()
   const isMobile = useIsMobile()
   const dispatch = useDispatch()
+  // Select only once per mount
+  const uiErrorCode = useSelector(getUiErrorCode, () => true)
```
Comment (dharit-tan) [inline packages/web/src/pages/something-wrong/SomethingWrong.tsx:36]: is the equality fn needed?

**Review by dharit-tan — APPROVED** [review]

File: packages/web/src/pages/something-wrong/SomethingWrong.tsx (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/pages/something-wrong/SomethingWrong.tsx:36]: without it, further errors could change the UI error code and make the emoji flicker/change. This equality makes it so it only uses the emoji error code on first render.

**Pull request merged**